### DISCUSSION TITLE
Mission "killcrawlerswarmlarge2" doesn't have a sonar label

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#13758] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Swarm spawns normally, but sonar marker "terminate swarm" is missing

### Reproduction steps

unlockmission killcrawlerswarmlarge2

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/32182507172a5ed48fb3ff625a9ac8394cdf6771 (commit in our private repo). Fix coming up in the next unstable build!
